README.md

```diff
@@ -477,6 +435,7 @@ The Determinate Nix installer has numerous advantages:
 * supporting a expanded test suite including 'curing' cases
 * supporting SELinux and OSTree based distributions without asking users to make compromises
 * operating as a single, static binary with external dependencies such as `openssl`, only calling existing system tools (like `useradd`) where necessary
+* supports remote building out of the box
```

Comment:
This is a very niche feature fixing a 'bug' on Macs. I don't think it's worth mentioning here.

```suggestion
```

If you wish to keep it please remove `out of the box`:

```suggestion
* As a MacOS remote build target, ensures `nix` is not absent from path
```

Reply:
I think this is interesting because its explicitly better in this way compared to the upstream installer. But I'm not married to it, and I think your rewrite is good. However, it does also support this behavior for Ubuntu and other targets, so I don't think we need to be too specific here.